packages/native/src/components/LinearGradient.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import { StyleProp, ViewStyle } from "react-native";
 import { LinearGradient } from "expo-linear-gradient";
-import { extractStyles } from "../utilities";
+// import { extractStyles } from "../../../core/src/utilities";
```

Comment:
if you need this file, just copy it over into utilities to make it easy for yourself. It's ok

Reply:
@peter I don't know if I need this file.  You did not use this option for other native files.

Reply:
@peterpme please advise ... should I use `extractStyles` here too ?

Reply:
If you don't need to extract the view styles from linear gradient then you can pass down the props regularly set style={style}

We extract the view styles on components when there's children components that need separate props.

Reply:
@peterpme Damn it!  I just added it.

Reply:
@peter I'll let you make the final call on the review ... keep / leave ...